### PR TITLE
MGMT-10081: Log install progress

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

**Comment by nmagnezi** [comment]
Print looks okay. Example from R2-D2 logs:

2022-04-24 09:44:44,710 - root - INFO - 140372922790144 - cluster 1a4b91e0-c325-4d10-a4ee-9f8b29e86745 progress info: {'total_percentage': 87, 'preparing_for_installation_stage_percentage': 100, 'installing_stage_percentage': 100, 'finalizing_stage_percentage': 50}

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py [hunk 1]
```diff
@@ -734,6 +734,7 @@ def wait_for_operators_to_finish(self, timeout=consts.CLUSTER_INSTALLATION_TIMEO
         else:
             statuses = [consts.OperatorStatus.AVAILABLE, consts.OperatorStatus.FAILED]
 
+        log.info("starting to wait for builtin operators")
```
Comment (eliorerz) [inline src/assisted_test_infra/test_infra/helper_classes/cluster.py:737]:
```suggestion
        log.info("Starting to wait for builtin operators")
```

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py [hunk 2]
```diff
@@ -743,6 +744,7 @@ def wait_for_operators_to_finish(self, timeout=consts.CLUSTER_INSTALLATION_TIMEO
             timeout=timeout,
             fall_on_error_status=False,
         )
+        log.info("starting to wait for OLM operators")
```
Comment (eliorerz) [inline src/assisted_test_infra/test_infra/helper_classes/cluster.py:747]:
```suggestion
        log.info("Starting to wait for OLM operators")
```

File: src/assisted_test_infra/test_infra/utils/operators_utils.py [hunk 3]
```diff
@@ -34,6 +39,9 @@ def _are_operators_in_status(
                 _Exception = consts.olm_operators.get_exception_factory(operator.name)
                 raise _Exception(f"Operator {operator.name} status is failed with info {operator.status_info}")
 
+    cluster = client.cluster_get(cluster_id=cluster_id).to_dict()
+    log.info("cluster %s progress info: %s", cluster_id, cluster["progress"])
+    log.debug("cluster %s info: %s", cluster_id, cluster)
```
Comment (eliorerz) [inline src/assisted_test_infra/test_infra/utils/operators_utils.py:44]:
```suggestion
    log.info("Cluster %s progress info: %s", cluster_id, cluster["progress"])
    log.debug("Cluster %s info: %s", cluster_id, cluster)
```

**Review by eliorerz — CHANGES_REQUESTED** [review]

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py (hunk 1 above)
Comment (nmagnezi) [reply src/assisted_test_infra/test_infra/helper_classes/cluster.py]: Done

File: src/assisted_test_infra/test_infra/utils/operators_utils.py (hunk 3 above)
Comment (nmagnezi) [reply src/assisted_test_infra/test_infra/utils/operators_utils.py]: Done

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py (hunk 2 above)
Comment (nmagnezi) [reply src/assisted_test_infra/test_infra/helper_classes/cluster.py]: Done

**Review by eliorerz — APPROVED** [review]
LGTM

**Pull request merged**